packages/fast-documentation/package.json

```diff
@@ -0,0 +1,130 @@
+{
+  "name": "fast-documentation",
+  "version": "0.1.0",
+  "private": true,
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "build": "tsc -p ./tsconfig.build.json",
+    "build:app": "webpack --progress --mode=production",
+    "clean:dist": "node ./build/clean.js dist",
+    "prepare": "npm run clean:dist && npm run build",
+    "prettier": "prettier --config ./.prettierrc --write \"**/*.{ts,tsx}\"",
+    "prettier:diff": "prettier --config ./.prettierrc \"**/*.{ts,tsx}\" --list-different",
+    "start": "webpack-dev-server",
+    "test": "npm run tslint && npm run unit-tests",
+    "tslint": "tslint -c ./tslint.json '**/*.ts{,x}'",
+    "tslint:fix": "tslint -c ./tslint.json --fix '**/*.ts{,x}'",
+    "unit-tests": "jest --maxWorkers=2 --passWithNoTests",
+    "unit-tests:watch": "jest --watch"
```

Comment:
It doesn't look like there are any tests being added here, can we remove them or should we add tests?